csv-extractor/csv_extractor.py

```diff
@@ -78,7 +80,7 @@ def create_data_points(values, timestamps):
     data_points = []
 
     for i, value_string in enumerate(values):
-        if pandas.notnull(value_string):
+        if value_string != '':
```

Comment:
pandas.notnull has more checks that might be important here.
I still think we should get rid of it, but we must replicate the ones that are relevant. Maybe check also for None, "null", "NULL"?

Reply:
Everything parsed by csv is stored in a string. Empty/None values are always parsed as empty string. I can confirm and make these changes; however, I think this check is sufficient.

Reply:
With pandas 1,5 would be parsed as a string, while 1 would be parsed to a number. If the csv reader only returns string, that means we can simplify some parts of the following lines.

If everything is a string, and we don't expect "null" to be used, or "NaN", then the best option is just "if not value_string:" imo.

Reply:
I am not sure if I understood you correctly, but I made changes here and looks much better now, thanks 👍